selfdrive/car/honda/carcontroller.py

```diff
@@ -127,7 +127,10 @@ def update(self, CC, CS, now_nanos):
     actuators = CC.actuators
     hud_control = CC.hudControl
     conversion = hondacan.get_cruise_speed_conversion(self.CP.carFingerprint, CS.is_metric)
-    hud_v_cruise = hud_control.setSpeed / conversion if hud_control.speedVisible else 255
+    if self.CP.carFingerprint in HONDA_BOSCH_RADARLESS:
+      hud_v_cruise = hud_control.setSpeed / conversion if hud_control.speedVisible and not CC.cruiseControl.override else 255
+    else:
+      hud_v_cruise = hud_control.setSpeed / conversion if hud_control.speedVisible else 255
```

Comment:
Alternative one liner:
`hud_v_cruise = hud_control.setSpeed / conversion if hud_control.speedVisible and ((self.CP.carFingerprint in HONDA_BOSCH_RADARLESS and not CC.cruiseControl.override) or self.CP.carFingerprint not in HONDA_BOSCH_RADARLESS) else 255`

Reply:
I would pass override into the ACC function like you did before, that's a bit cleaner